csv lesen + schreiben sollte gefixed sein

Yasamato opened the pull request:
todo: wahl.html und dashboard anpassen

Commit:
moved directory

Commit:
Forgot....

Commit:
fixed csv lesen+schreiben

tintinmaster commented on web/html/projektErstellung.html:
```diff
@@ -17,8 +17,7 @@ <h1>Projekt-Einschreibe-Formular</h1>
 				<label for="inputProjektname">Projektname</label>
-				<input type="text" class="form-control" id="inputProjektname" placeholder="Projektname" required name="inputProjektname"
-				 autofocus>
+				<input type="text" class="form-control" id="inputProjektname" placeholder="Projektname" required name="pName" autofocus>
```
Haben die name-Tags Probleme mit den identisch lautenden ID's gemacht und du hast sie deshalb geändert, oder weil sie so lang sind?

Yasamato replied:
zu lang, finde die bezeichnung als input** vor jedem auch nicht grad das schönste

tintinmaster replied:
Okay :D

tintinmaster commented on web/php/login.php:
```diff
@@ -40,11 +40,22 @@ function ldap_authenticate() {
+			"nachname" => "wie arwin"
+		];
+	}
+	elseif($_POST['user'] == "wasd"){
```
Warum so ein sinnloser Benutzername ? schueler oder student ist viel passender @Agent77326

Yasamato replied:
ja, hab es zu Schüler geändert im nächsten Submit

tintinmaster replied:
Perfekt

tintinmaster commented on web/php/login.php:
```diff
@@ -40,11 +40,22 @@ function ldap_authenticate() {
+		$_SESSION['benutzer'] = [
+			"uid" => "testeeeee",
+			"typ" => "students",
+			"klasse" => $_POST['pw'] . "aaa",
```
Sehr gute Idee, die Klassenstufe als Passwort zum Testen zu übergeben 👍

Commit:
Done wahl

Wahl.html hat keine probleme mehr

Commit:
moved dashboard -> html/

Commit:
moved dashboard

funktioniert wieder (anzeige), php-backend on form submit noch nicht getan, sowie html buttons-bug nicht gefixt
